frontend/storybook/src/components/entry/admin/images/meta/pages/confirmation.ts

```diff
@@ -0,0 +1,32 @@
+import "@elements/entry/admin/images/meta/page";
+import "@elements/core/titles/variants/underlined-title";
+import "@elements/entry/admin/images/add/add-image";
+import { Rectangle } from "~/components/core/buttons/rectangle";
+
+export default {
+  title: 'Entry/Admin/Images/Meta/Pages',
+}
+
+const STR_LABEL = "Label Images";
```

Comment:
I think this could be better designed as an element, but not going to block the PR on that